bucket/bzip2.json

```diff
@@ -21,5 +22,15 @@
     "pre_install": "cp \"$dir\\bzip2.exe\" \"$dir\\bunzip2.exe\"; cp \"$dir\\bzip2.exe\" \"$dir\\bzcat.exe\"",
```

Comment:
Add binaries for this:

bin: [bzip2.exe, [bzip2.exe, bzcat], [bzip2.exe, bunzip2]]

Reply:
Current bin:

```json
    "bin": [
        "bunzip2.exe",
        "bzcat.exe",
        "bzip2.exe",
        "bzip2recover.exe"
    ]
```

What is your proposal?

Reply:
You do not see problem when you copy same file twice with different name. 

That is nonsense. 

Just use that as i wrote

```yml
bin:
  - bzip2.exe
  - - bzip2.exe
    - bzcat
  - - bzip2.exe
    - bunzip2
```

_Too lazy to quote json_